src/components/profiles-modals/ImportProfileModal.vue

```diff
@@ -317,32 +306,83 @@ export default class ImportProfileModal extends mixins(ProfilesMixin) {
             <h2 class="modal-title">Packages to be installed</h2>
         </template>
         <template v-slot:body>
-            <OnlineModList :paged-mod-list="profileToOnlineMods" :read-only="true" />
+            <OnlineModList :paged-mod-list="profileMods.known" :read-only="true" />
+            <div v-if="!profileMods.known.length || profileMods.unknown.length" class="notification is-warning margin-top">
+                <p v-if="!profileMods.known.length">
+                    None of the packages in the profile were found on Thunderstore:
+                </p>
+                <p v-else>
+                    Some of the packages in the profile were not found on Thunderstore:
+                </p>
+
+                <p class="margin-top">{{ profileMods.unknown.join(", ") }}</p>
+
+                <p v-if="profileMods.known.length" class="margin-top">
+                    These packages will not be installed.
+                </p>
+                <p v-else class="margin-top">
+                    Ensure the profile is intended for the currently selected game.
+                </p>
```

Comment:
This is probably fairly useless as an indicator since the BepInEx/MelonLoader packages can be shared across games.

Would it be better UX if we have another modal step, and in that step we re-show the game name and ask if it's correct?